### PR TITLE
Remove duplicate step list

### PR DESCRIPTION
**Pull request by toabctl**
Commit e7355ff4c910af28b428 added some help instructions when running
'run.sh' but it also duplicated the list of possible actions (steps).
Remove that duplicate list but still allow to see the help if
requested.

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -21,13 +21,15 @@ socok8s_absolute_dir="$( cd "$(dirname "$0")" ; pwd -P )"
...
-    case "$deployment_action" in
-      -h|--help) usage ; exit 0 ;;
-       *) source ${scripts_absolute_dir}/pre-flight-checks.sh "validate_cli_options $1"
-          break ;;
-    esac
-done
+# be able to show the help. Later we have the same case (which includes
+# the help) but the pre-flight checks might already abort before
+# being able to show the help
+case "$deployment_action" in
```
Comment (jsuchome) [inline run.sh]: Why `case` for just one option?

Comment (toabctl) [reply run.sh]: fixed

**Comment by toabctl** [comment]
The downside of this PR is, that if you run a non-valid deployment action (eg `./run.sh xyz`) then you get the output from the preflight checks and other stuff before you actually see the `Invalid option foo, Check --help for valid options` message.

**Comment by rajdharwadkar** [comment]
> The downside of this PR is, that if you run a non-valid deployment action (eg `./run.sh xyz`) then you get the output from the preflight checks and other stuff before you actually see the `Invalid option foo, Check --help for valid options` message.

Yes thats reason this is duplicated . User expects validation right at beginning not after seeing those output from source files.

**Comment by Itxaka** [comment]
Lets close this for now, it can be restored later if needed

**Pull request closed without merging**